src/simulator/scenes/gate/gateFront.xml

```diff
@@ -293,43 +299,216 @@
       <y>0</y>
     </orientation>
     <offsetp>
-      <x>-1.5</x>
-      <y>-3.0</y>
+      <x>-0</x>
+      <y>-0</y>
       <z>0</z>
     </offsetp>
     <offsetr>
-      <x> 3.1415</x>
+      <x> 3.14</x>
```

Comment:
Why did we drop precision here?

Reply:
.001 radians is .05 degrees, I just stuck to the precision they used initially